api-go/tools/puppeteer/index.js

```diff
@@ -7,14 +7,21 @@
 const echarts = require("echarts");
 const { createCanvas } = require('canvas');
 const jsdom = require("jsdom");
-const { fetchContributorsData, fetchMonthlyData } = require('./fetch');
+const { fetchContributorsData, fetchMonthlyData, fetchMergeContributor } = require('./fetch');
 const { updateSeries } = require('./utils');
 const { JSDOM } = jsdom;
 
 const config = {
   width: 896,
   height: 550,
-}
+};
+
+const mergeRepoList = [
```

Comment:
Could we could maintain a json or yaml file to store this configuration?

Reply:
Both config and mergeRepoList ? @juzhiyuan 

I found [js-yaml](https://www.npmjs.com/package/js-yaml), after trying it feels good, we can use the YAML file.